### DISCUSSION TITLE
Custom classes to checkout fields

### DISCUSSION THREAD

**Discussion by wdmarketLV** [breezefront/community#124] [1. Help]
Hello, I am using firecheckout. 
I added a custom address field. 
I need to style it, but I cant, because it doesn't have a class, all other fields is possible, 
what I need to do to add class checkout field? 
Regards

![image](https://user-images.githubusercontent.com/79901550/236498855-df37671e-41b7-45c3-8b4f-faa2a8edf6c5.png)

**Comment by roma84** [comment]
This question is not related to Breeze.

You can use the CSS attribute selector:

```css
input[name="company"] {
    //your styles here
}
```
